static/js/Misc.jsx

```diff
@@ -3288,11 +3288,17 @@ const Autocompleter = ({getSuggestions, showSuggestionsOnSelect, inputPlaceholde
     )
 }
 
+const imgAltTextFactory = (caption) => {
+  const lang = Sefaria.interfaceLang === "hebrew" ? 'he' : 'en';
+  const fallBackMessage = lang === 'he' ? "Illustrative image" : "תמונה להמחשה";
+  const captionMessage = caption[lang];
+  if(captionMessage) {return captionMessage} else {return fallBackMessage}
+}
 const ImageWithCaption = ({photoLink, caption }) => {
-  
+  const altText = imgAltTextFactory(caption)
   return (
     <div>
-        <img class="imageWithCaptionPhoto" src={photoLink}/>
+        <img class="imageWithCaptionPhoto" src={photoLink} alt={altText}/>
```

Comment:
Since the function call is quite short, I think it's better to put it inline here as opposed to creating a local var.